library/src/main/kotlin/app/futured/donut/DonutProgressView.kt

```diff
@@ -368,10 +493,27 @@ class DonutProgressView @JvmOverloads constructor(
     }
 
     override fun onDraw(canvas: Canvas) {
-        super.onDraw(canvas)
         canvas.translate(centerX, centerY)
 
         bgLine.draw(canvas)
         lines.forEach { it.draw(canvas) }
+
+        if (DEBUG_COLLISIONS) {
+            drawDebugTapCollisions(canvas)
+        }
+    }
+
+    private fun drawDebugTapCollisions(canvas: Canvas) {
+        canvas.translate(-centerX, -centerY)
+        collisionRects.forEach {
+            canvas.drawRect(it.second, redPaint)
+        }
+        tapRect?.let { canvas.drawRect(it, blackPaint) }
+    }
+
+    private fun d(message: () -> String) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/961d58caf19d5189c8ff62ff1a0576433dabdd98/library/src/main/kotlin/app/futured/donut/DonutProgressView.kt#L514"></span>Function names should be at least 3 characters long.</td>
    </tr>
  </tbody>
</table>